EIPS/eip-7699.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 7699
```

Comment:
```suggestion
eip: 7XXX
```
Numbers are issued by editors and associates.  Once a valid PR is created in the ERC repo a number will be manually issued.